express/src/views/no-account-found.njk

```diff
@@ -2,12 +2,12 @@
 
 {% set headerActiveItem = "sign-in" %}
 {% set pageTitle = "No account found" %}
-{% set backLinkPath = "/sign-in-password" %}
+{% set backLinkPath = "/sign-in/enter-password" %}
 
 {% block mainContent %}
   <h1 class="govuk-heading-l">No account found</h1>
   <p class="govuk-body">We did not find an account with those details.</p>
-  <p class="govuk-body"><a class="govuk-link" href="/sign-in">Try signing in again</a> or
-    <a class="govuk-link" href="/create/get-email">create an account</a>.
+  <p class="govuk-body"><a class="govuk-link" href="/sign-in/enter-email-address">Try signing in again</a> or
+    <a class="govuk-link" href="/register/enter-email-address">create an account</a>.
```

Comment:
```suggestion
    <a class="govuk-link" href="/register">create an account</a>.
```
And create a route to redirect GET `/register` to `/register/enter-email-address`